tests/unit_tests/dft/include/compute_tester.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef ONEMKL_COMPUTE_TESTER_HPP
+#define ONEMKL_COMPUTE_TESTER_HPP
+
+#if __has_include(<sycl/sycl.hpp>)
+#include <sycl/sycl.hpp>
+#else
+#include <CL/sycl.hpp>
+#endif
+#include "oneapi/mkl.hpp"
+#include "test_helper.hpp"
+#include "../../blas/include/test_common.hpp"
+#include "test_common.hpp"
+#include "oneapi/mkl/dft/descriptor.hpp"
+#include "oneapi/mkl/dft/types.hpp"
+#include "reference_dft.hpp"
+
+extern std::vector<sycl::device *> devices;
+
+namespace {
+using std::vector;
+using namespace oneapi::mkl;
+using namespace sycl;
+
+//constexpr size_t default_1d_lengths = 6;
+constexpr size_t default_1d_lengths = 137;
```

Comment:
I'm not sure what we're showing by testing with different compute sizes?

It might be good to have a default size that is a power of 2 instead of 137. FFT algorithms are usually implemented for powers of 2, then extended to have factors that are other primes.

Reply:
Different kernels can be used for different sizes. As you said the power of 2 sizes are the most efficient ones but you need to handle the remainder in a different way. Also large sizes won't be handled the same way as small sizes.